### PR TITLE
feat(ignorePaths): ignore CAD files and vector files

### PR DESCRIPTION
**Pull request by UMiho**
I added .dxf and .svg, both are image file format.
spell check fail log: https://github.com/tier4/pilot-auto-documentation/actions/runs/2515295557
Screenshot from 2022-06-17 21-12-57

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -17,6 +18,7 @@
     "**/*.pcd",
     "**/*.pt",
     "**/*.stl",
+    "**/*.svg",
```
Comment (kenji-miyake) [inline .cspell.json:21]: We don't ignore SVG files, instead, add some patterns to ignore.
https://github.com/tier4/autoware-spell-check-dict/blob/29c6362fdbd20e379db1763fc6168c4744558f86/.cspell.json#L153-L164

```suggestion
```

Comment (UMiho) [reply .cspell.json]: Got it. I'll add some RegExp to pass it.

File: .cspell.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "language": "en",
   "allowCompoundWords": false,
   "ignorePaths": [
+    "**.*.dxf",
```
Comment (kenji-miyake) [inline .cspell.json:6]: Is this correct? :thinking: 

```suggestion
    "**/*.dxf",
```

Comment (UMiho) [reply .cspell.json:6]: Sorry, that's typo.

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**